databases/sql-1.md

```diff
@@ -54,15 +54,18 @@ Most software applications store data of some sort -- user data, content, etc. D
 #### Section 1: Creating tables and basic querying
 
 1. Work through the [Codecademy SQL Tutorial](https://www.codecademy.com/learn/learn-sql) **Section 1 (Manipulation)**:
-2. SQLite is a version of SQL that comes installed on mac! Try it out:
-   - In Terminal, type `sqlite3 food.sqlite3` to use SQLite. This command will open sqlite and save your database to a file named `food.sqlite3`.
+
+2. PostgreSQL is a popular, open-source version of SQL. Try it out:
+   - In Terminal, type `pg_dump foo > food.psql`. This commande will save your database to a file names `food.psql`.
```

Comment:
The equivalent for postgres is to create a new database. IIRC there should already be one named after their username which is fine for this exercise so we can probably just dive in.